### PR TITLE
tcl string format

### PR DESCRIPTION
**Pull request by alain-rs**
> ### Motivate of the pull request
> - [x ] To address an existing issue. If so, please provide a link to the issue: <issue id>
> - [ ] Breaking new feature. If so, please describe details in the description part.

> ### Describe the technical details
> #### What is currently done? (Provide issue link if applicable)
> 
> 
> 
> 
> 
>
> #### What does this pull request change?
> 
> 
> 
> 
> 

> ### Which part of the code base require a change
> 
> - [ ] Library: <Specify the library name>
> - [ ] Plug-in: <Specify the plugin name>
> - [ ] Engine
> - [ ] Documentation
> - [ ] Regression tests
> - [ ] Continous Integration (CI) scripts

> ### Impact of the pull request

> - [ ] Require a change on Quality of Results (QoR)
> - [ ] Break back-compatibility. If so, please list who may be influenced.

Make embedded tcl scripts readable

### CODE REVIEW

**Comment by alain-rs** [comment]
BTW, these tests don't seem to run

**Comment by KochynVolodymyr** [comment]
> BTW, these tests don't seem to run

I see this:
TEST_LOOP_ENTERED
CONSOLE GUI: pwd command
SUCCESS
CONSOLE GUI: source command
SUCCESS
CONSOLE GUI: multiline
SUCCESS
GUI EXIT

File: src/Console/Test/console_test.cpp [hunk 1]
```diff
@@ -54,8 +54,14 @@ TCL_TEST(console_proc) {
 
 TCL_TEST(console_multiline) {
   FOEDAG::TclConsoleWidget *console = FOEDAG::InitConsole(interp);
-  QString res = console->getPrompt() + "proc test {} {\nputs test\n}\n" +
-                "test\n" + console->getPrompt();
+  QString script =
+      R"("
+          proc test {} {
```
Comment (KochynVolodymyr) [inline src/Console/Test/console_test.cpp:59]: current implementation send every character. So it will not work since you put many spaces.

**Comment by alain-rs** [comment]
What is the Makefile target to trun this test:
make test
make test/gui
make test/batch 

do not run this test

**Comment by alain-rs** [comment]
make test/unittest does not run these tests either

**Comment by KochynVolodymyr** [comment]
> make test/unittest does not run these tests either

Everything is the same like it was before.

**Comment by KochynVolodymyr** [comment]
> You are not answering the question. How do I execute your new unit tests. What command?

make test/gui. Like before

**Pull request merged**